src/Sulu/Component/Content/Document/Subscriber/OrderSubscriber.php

```diff
@@ -64,19 +73,18 @@ public function handlePersist(PersistEvent $event)
 
         $propertyName = $this->encoder->systemName(self::FIELD);
 
-        if ($node->hasProperty($propertyName)) {
-            return;
-        }
-
         $parent = $node->getParent();
-        $nodeCount = count($parent->getNodes());
-        $order = ($nodeCount + 1) * 10;
+        $count = 0;
+        foreach ($parent->getNodes() as $childNode) {
```

Comment:
To be honest, without looking at this closer I could not say. I would
vote to do it "manually" now rather than refactoring the core and
falling into a time hole.

On Thu, Jul 16, 2015 at 07:36:47AM -0700, Daniel Rotter wrote:
>    In [1]src/Sulu/Component/Content/Document/Subscriber/OrderSubscriber.php:
> 
>  >          $parent = $node->getParent();
>  > -        $nodeCount = count($parent->getNodes());
>  > -        $order = ($nodeCount + 1) * 10;
>  > +        $count = 0;
>  > +        foreach ($parent->getNodes() as $childNode) {
> 
>    I guess it is always more efficient, so I would do the same for the
>    ReorderEvent, especially because it is very confusing if the way to get
>    the node changes all the time.
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/1378#discussion_r34793821
>    2. https://github.com/sulu-io/sulu/pull/1378/files#r34793821
>    3. https://github.com/sulu-io/sulu/pull/1378/files#r34793821